charts/plugin-fees/templates/fees/configmap.yaml

```
@@ -7,6 +7,7 @@ metadata:
 data:
   # -- Default Environment variables for the plugin-fees 
   ENV_NAME: {{ .Values.fees.configmap.ENV_NAME | default "development" | quote }}
+  APPLICATION_NAME: {{ .Values.fees.configmap.APPLICATION_NAME | default "plugin-fees" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)